modular_skyrat/modules/reagent_forging/code/forge_weapons.dm

```diff
@@ -240,3 +240,68 @@
 /obj/item/shovel/reagent_weapon/Initialize(mapload)
 	. = ..()
 	AddComponent(/datum/component/reagent_weapon)
+
+/obj/item/ammo_casing/caseless/arrow/wood/forged
+	desc = "An arrow made of wood, typically fired from a bow. It can be reinforced with sinew."
+	projectile_type = /obj/projectile/bullet/reusable/arrow/wood/forged
+
+/obj/item/ammo_casing/caseless/arrow/wood/forged/attackby(obj/item/attacking_item, mob/user, params)
+	if(istype(attacking_item, /obj/item/stack/sheet/sinew))
+		var/obj/item/stack/stack_item = attacking_item
+		if(!stack_item.use(1))
+			return
+		new /obj/item/ammo_casing/caseless/arrow/ash(get_turf(src))
+		qdel(src)
+		return
+	return ..()
+
+/obj/projectile/bullet/reusable/arrow/wood/forged
+	name = "wooden arrow"
+	desc = "Woosh!"
+	damage = 25
+	icon_state = "arrow"
+	ammo_type = /obj/item/ammo_casing/caseless/arrow/wood/forged
+
+/obj/item/forging/reagent_weapon/bokken
+	name = "reagent bokken"
+	desc = "A bokken that can be imbued with a reagent. It can be dual-wielded to increase block chance!"
+	force = 15
```

Comment:
Make it 12, maybe? 15 damage with 40% block is pretty high.

Reply:
we can see how it goes